scvi/external/cellassign/_module.py

```diff
@@ -114,7 +113,7 @@ def _get_generative_input(self, tensors, inference_outputs):
 
         to_cat = []
         if self.n_batch > 0:
-            to_cat.append(one_hot(tensors[REGISTRY_KEYS.BATCH_KEY], self.n_batch))
+            to_cat.append(F.one_hot(tensors[REGISTRY_KEYS.BATCH_KEY], self.n_batch).squeeze(-2))
```

Comment:
```suggestion
            to_cat.append(F.one_hot(tensors[REGISTRY_KEYS.BATCH_KEY].squeeze(-1), self.n_batch)
```
Could we actually call `.squeeze(-1)` on `tensors[REGISTRY_KEYS.BATCH_KEY]` instead of `squeeze(-2)` (for all instances of this)? I think it makes it a bit more clear.

Reply:
Oh this is a much better idea – thanks! Updated.